api/api/spec/spec.yaml

```diff
@@ -5373,6 +5373,29 @@ components:
       schema:
         type: string
         format: alphanumeric
+    vulnerability_field:
+      in: path
+      name: field
+      description: "Vulnerability inventory field"
+      required: true
+      schema:
+        type: string
+        enum:
+        - cve
+        - name
+        - version
+        - architecture
+        - detection_time
+        - severity
+        - cvss2_score
+        - cvss3_score
+        - references
```

Comment:
Based on https://github.com/wazuh/wazuh/issues/12619, the `references` field is finally named `external_references`.

Reply:
This development was finished before that field changed. Fixed.